File: Ad-Hoc/URI 1893.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val inicio = leitor.nextInt()
        val finau = leitor.nextInt()
        if (inicio >= 0 && finau <= 2) {
            println("nova")
        } else if (finau > inicio && finau <= 96) {
            println("crescente")
        } else if (inicio >= finau && finau <= 96) {
            println("minguante")
        } else { 
            println("cheia")
        }
    }
    
}

File: Ad-Hoc/URI 1893.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    val inicio: Int = leitor.nextInt();
    val final: Int = leitor.nextInt();
    if (inicio >= 0 && final <= 2) {
        println("nova");
    } else if (final > inicio && final <= 96) {
        println("crescente");
    } else if (inicio >= final && final <= 96) {
        println("minguante");
    } else { 
        println("cheia");
    }
}